neusomatic/python/utils.py

```diff
@@ -93,3 +118,85 @@ def prob2phred(p, max_phred=100):
         if Q > max_phred:
             Q = max_phred
     return Q
+
+
+def write_tsv_file(tsv_file, records, sep='\t', add_fields=[]):
+    with open(tsv_file, "w") as f_o:
+        for x in records:
+            f_o.write(sep.join(map(str, x + add_fields)) + "\n")
+
+
+def read_tsv_file(tsv_file, sep='\t', fields=None):
+    records = []
+    with open(tsv_file) as i_f:
+        for line in i_f:
+            if not line.strip():
+                continue
+            x = line.strip().split(sep)
+            if fields is not None:
+                x = [x[i] for i in fields]
+            records.append(x)
+    return records
+
+
+def vcf_2_bed(vcf_file, bed_file, add_fields=[]):
+    with open(bed_file, "w") as f_o:
```

Comment:
You can simply do 'with open(bed_file, "w") as f_o, open(vcf_file, "r") as f_i` instead of nested `with open`

Reply:
@marghoob  done